### ISSUE TITLE
Accessibility: GiphyGridController cells are inaccessible while using VoiceOver

### ISSUE THREAD

**Issue by 3arrett** [Giphy/giphy-ios-sdk#82]
Using the latest version `1.3.0` and the `GiphyGridController` embedded. 

While using the VoiceOver, neither the Cells nor the UICollectionView is accessible. You can tap a cell and it will read presumably the fileName but gives no focus and you can't scroll the collection view either.

**Comment by cgmaier** [comment]
hey @3arrett, would love to help with this. do you have any pointers on how to achieve this behavior? 

is it just a matter of exposing / documenting accessibility labels for the UICollectionView and cells?

**Comment by 3arrett** [comment]
Yeah, by default `UICollectionViewCell`s aren't an accessibility element and infer their contents if any. Since the only content in the `GPHMediaCell` is the ImageView, it why tapping it reads off the file name.

At a minimum you can just add `isAccessibilityElement = true` on the `GPHMediaCell` itself. Not knowing the full API, if you have a relevant name for the cell you can add that as the `accessibilityLabel` itself. Otherwise if you're just ending up being generic, it'd always be good to announce it's position within the datasource, e.g. `GIF 3 of 700`

There shouldn't be much more than that because once you have cells accessible, you are now able to triple swipe up and down to page through all the elements.

**Comment by cgmaier** [comment]
hey @3arrett I hope to get to this with the next release. 

I found some documentation for how to test this behavior here: 

https://developer.apple.com/library/archive/technotes/TestingAccessibilityOfiOSApps/TestAccessibilityonYourDevicewithVoiceOver/TestAccessibilityonYourDevicewithVoiceOver.html

feel free to share any other resources you think may be useful : )

**Comment by 3arrett** [comment]
Great! A great resource also to take look into is the WWDC videos. Here are some of the quick overview ones you can take a look into to:

- Design For Everyone - WWDC 2017 - https://developer.apple.com/videos/play/wwdc2017/806
- VoiceOver: App Testing Beyond The Visuals - WWDC 2018 - https://developer.apple.com/videos/play/wwdc2018/226
- Deliver an Exceptional Accessibility Experience - WWDC 2018 - https://developer.apple.com/videos/play/wwdc2018/230

**Comment by Lizmaddy** [comment]
@cgmaier - Any update on resolving this issue?

**Comment by cgmaier** [comment]
hey everyone, as of `v2.1.9` GIPHY iOS SDK supports VoiceOver.

We added accessibility information for our `GiphyGridController` cells - every cell has descriptive text for VoiceOver to read by one-finger tap, and it’s possible to triple-swipe up and down to page through all the elements with VoiceOver on.
